.github/workflows/test-full-sync.yml

```diff
@@ -79,7 +79,6 @@ jobs:
 
       # Setup Docker Buildx to allow use of docker cache layers from GH
       - name: Set up Docker Buildx
-        id: buildx
```

Comment:
This is a future proof id needed for arm64 builds. I'd recommend to keep it

Reply:
I'll undo for now